doc/api.rst

```diff
@@ -34,4 +37,30 @@ Graph Protocols
    :toctree: generated/
 
    GraphProtocol
-   EquivalenceClassProtocol
\ No newline at end of file
+   EquivalenceClassProtocol
+   
+Constraint-Based Structure Learning
```

Comment:
Shouldn't these have gone in the previous PR?  Perhaps they can go into a separate PR?